Recommend line-awesome add-on for V24 (#2477)
vaadin/docs: articles/upgrading/index.adoc

// Allow CheckboxGroup
pass:[<!-- vale Vaadin.ProductName = NO -->]

== Line-Awesome Icons Set Library

Vaadin 23 starter projects used https://icons8.com/line-awesome[`line-awesome`] icons set library.

> You may need `line-awesome` library in your project as well.

Reviewer: ```suggestion
An older project based on a starter may need the `line-awesome` library as well.
```

> If so, we recommend to use Vaadin https://vaadin.com/directory/component/line-awesome-icons-for-vaadin[Line-Awesome] add-on with Vaadin 24.

Reviewer: ```suggestion
In these cases it is recommend to use the Vaadin https://vaadin.com/directory/component/line-awesome-icons-for-vaadin[Line-Awesome] add-on with Vaadin 24.
```

> In Vaadin 24 this add-on doesn't require a new bundle building, so your app would benefit from pre-compiled bundle feature.

Reviewer: ```suggestion
In Vaadin 24 this add-on doesn't require building a new bundle, so your application would then benefit from pre-compiled bundle feature.
```
